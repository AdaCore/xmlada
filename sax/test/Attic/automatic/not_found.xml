<?xml version="1.0"?>
<!DOCTYPE datasource SYSTEM "not_existDTD">

<root>
</root>

