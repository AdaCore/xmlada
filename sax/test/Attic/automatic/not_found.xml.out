Sax.Set_Document_Locator ()
Sax.Start_Document ()
Sax.Ignorable_Whitespace ( 10, 1) at not_found.xml:2:2
Sax.Start_DTD (datasource, , not_existDTD) at not_found.xml:3:1
Sax.Error (not_found.xml:2:43: External subset not found: not_existDTD, at not_found.xml:3:1)
Sax.End_DTD () at not_found.xml:3:1
Sax.Ignorable_Whitespace ( 10 10, 2) at not_found.xml:4:2
Sax.Start_Element (, root, root) at not_found.xml:5:1
Sax.Ignorable_Whitespace ( 10, 1) at not_found.xml:5:2
Sax.End_Element (, root, root) at not_found.xml:6:1
Sax.Ignorable_Whitespace ( 10 10, 2) at not_found.xml:7:1
Sax.End_Prefix_Mapping (xml) at not_found.xml:7:1
Sax.End_Document ()
